tools/pipelines/templates/build-npm-package.yml

```diff
@@ -277,6 +281,33 @@ stages:
               tagName: ${{ parameters.tagName }}
               interdependencyRange: ${{ parameters.interdependencyRange }}
 
+        # Generate rollups
+        - ${{ if or(eq(parameters.releaseBuildOverride, 'prerelease - alpha'), eq(parameters.releaseBuildOverride, 'prerelease - beta')) }}:
+          - task: Npm@1
+            displayName: npm run ${{ parameters.taskGenerateRollups }}
+            inputs: 
+              command: 'custom'
+              workingDir: ${{ parameters.buildDirectory }}
+              customCommand: 'run ${{ parameters.taskGenerateRollups }}'
+        
+        # Update package.json types to alpha
+        - ${{ if (eq(parameters.releaseBuildOverride, 'prerelease - alpha') }}:
+          - task: Npm@1
+            displayName: Set package.json types to alpha
+            inputs:
+              command: 'custom'
+              workingDir: ${{ parameters.buildDirectory }}
+              customCommand: 'pnpm exec flub release setDtsTypes -g ${{ parameters.tagName }} --types alpha --json | jq -c > alpha-types.json'
```

Comment:
You don't need `pnpm exec`. If you want to call flub directly, make this a bash step.

Reply:
And if you want to lock it to the local version, then you'll need to add a package.json script to the root and then call it in CI.